### ISSUE TITLE
Block breaking

### ISSUE THREAD

**Issue by mindcoding** [DragonetMC/Dragonet-Legacy#31]
Block breaking does not show on MCPE is there a way to fix this. Also is there a way to fix slow chat?

**Comment by Creeperface01** [comment]
Chat is fast i think. You can test it on our server 89.203.243.138:19132

**Comment by mindcoding** [comment]
And block breaking?

**Comment by DefinitlyEvil** [comment]
This is due to packet loss, network bad will cause that.

**Comment by mindcoding** [comment]
Is there an easy why to fix this?

**Comment by mindcoding** [comment]
All so I can't access the test server.

**Comment by wordandahalf** [comment]
@mindcoding Packet loss = bad WiFi, we can't fix your Internet

**Comment by Creeperface01** [comment]
:D
